src/lib.rs

```diff
@@ -128,20 +127,21 @@ impl Drop for DropBomb {
 ///
 /// [`std::task::Context`]: std::task::Context
 #[repr(C)]
+#[cfg_attr(feature = "sabi", derive(abi_stable::StableAbi))]
```

Comment:
After a quick glance of `abi_stable`, it seems the field name is also part of ABI. Introducing it would just pin everything things down, yes? If so, we should rethink all field names, eg. rename `waker_ref` -> `waker_ptr` for consistency and `phantom` -> `_marker`.

Reply:
I actually think `waker` would be fine as a name. It's obvious it's a reference and a pointer given the type. I agree that `phantom` should be  `_marker`.

Reply:
That's reasonable. `waker` should be enough.